### PR TITLE
Welcome page updates

### LINKED ISSUE #20: Standard welcome page contents
I want to determine what will be the content of our welcome pages. Layout stays as v2, rails-ex and dancer-ex.
The current page for rails-ex:
image

Issues:
1. **Web Console** section: I'm unsure the copy "add collaborator access authorizations, designate custom domain aliases, and manage domain memberships" relates to the web console in v3
2. **Command Line Tools** section: documentation is for v2
3. **Development Resources** section: except for StackOverflow and Git docs, all links point to v2 resources.
4. In **Deploying code changes**, the first thing we do is list the steps to use GitHub webhook, but that won't work if you are trying OpenShift on a non-GitHub accessible network, and it not totally required.
5. **Working in your local Git repository** links to v2 documentation
6. **Working in your local Git repository** steps won't work unless you change the template parameter to point to your fork of the github repo.

Content present in django-ex and not in rails-ex:
- Steps how the example project was created
- Dynamic request info: hostname of the pod serving the request, database connection info, page view counter
  image

---

@bparees @mfojtik @jhadvig @ewolinetz could you please comment on the points above?
Once I have a clearer picture of the contents we want in the welcome pages I can make PRs to update what's needed.

### PR DESCRIPTION
**Pull request by rhcarvalho**
More updates to come.

Fixes #20.

@bparees FYI

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -217,16 +218,20 @@
           <section class='col-xs-12 col-sm-6 col-md-6'>
             <section>
               <h2>Deploying code changes</h2>
```
Comment (rhcarvalho) [inline public/index.html:223]: @bparees if you fork the repo, clone and use the templates to deploy to OpenShift, you will actually deploy the code from openshift/rail-ex, and not your fork. Without changing the SOURCE_REPOSITORY_URL parameter, your webhooks will trigger rebuilds of openshift/rail-ex no matter what code changes you've made and pushed into your fork.

I think we need to address this by explaining this required step. What do you say? Where would it go?

Comment (bparees) [inline public/index.html:223]: it needs to be in general docs for using the templates, we aren't going to doc that in each and every sample app.

**Comment by rhcarvalho** [comment]
New look:
image

@bparees @ewolinetz @jhadvig PTAL

**Comment by rhcarvalho** [comment]
Included link to http://docs.openshift.org/latest/using_images/sti_images/ruby.html as we have in the Dancer example (for Perl)

**Pull request merged**